api/v1alpha1/releaseserviceconfig_types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const ReleaseServiceConfigResourceName string = "config"
+
+// ReleaseServiceConfigSpec defines the desired state of ReleaseServiceConfig.
+type ReleaseServiceConfigSpec struct {
+	// AllowLocalPipelines is the boolean that specifies whether or not pipelines stored locally
+	// on the cluster may be used
+	// +optional
+	AllowLocalPipelines bool `json:"allowLocalPipelines,omitempty"`
```

Comment:
What about to just call it debug and this being a side effect of the debug mode?

Reply:
switched to debug